composer.json

```diff
@@ -30,13 +30,12 @@
         "illuminate/routing": "5.5.*|5.6.*|5.7.*|5.8.*",
         "illuminate/support": "5.5.*|5.6.*|5.7.*|5.8.*",
         "illuminate/validation": "5.5.*|5.6.*|5.7.*|5.8.*",
-        "opis/closure": "^3.0",
-        "webonyx/graphql-php": "^0.13"
+        "webonyx/graphql-php": "^0.13.2"
```

Comment:
What feature are we requiring from 0.13.2 since you changed it? Isn't it better to have it as loose as possible, composer will automatically install the best version possible.

Reply:
https://github.com/webonyx/graphql-php/pull/475